build.gradle

```diff
@@ -39,6 +39,9 @@ plugins {
   id 'org.nosphere.apache.rat' version "0.8.1"
   id "io.swagger.core.v3.swagger-gradle-plugin" version "${swaggerVersion}"
 
+  // When updating the spotbugs gradle plugin, check if it already
+  // includes spotbugs version 4.7.4, in which case CVE-2022-42920 can
+  // be dropped from gradle/resources/dependencycheck-suppressions.xml
```

Comment:
Is this comment still needed? The spotbugs version is 5.1.3, right?

Reply:
not sure, why we added this comment recently: https://github.com/apache/kafka/pull/14476/files
anyhow, we need do to the cleanup in Apache Kafka, right?